templates/media/field--image.html.twig

```diff
@@ -2,51 +2,7 @@
 /**
  * @file
  * Theme override for all fields of type "image" (any etypes/bundles).
- * ~ Removed <div{{ attributes }}> from multiple section.
- * ~ Removed all <div{{ item.attributes }}> wrappers.
- * ~ Removed title_classes.
- *
- * Available variables:
- * - attributes: HTML attributes for the containing element.
- * - label_hidden: Whether to show the field label or not.
- * - title_attributes: HTML attributes for the title.
- * - label: The label for the field.
- * - multiple: TRUE if a field can contain multiple items.
- * - items: List of all the field items. Each item contains:
- *   - attributes: List of HTML attributes for each item.
- *   - content: The field item's content.
- * - entity_type: The entity type to which the field belongs.
- * - field_name: The name of the field.
- * - field_type: The type of the field.
- * - label_display: The display settings for the label.
- *
- * @see field.html.twig
- *
- * @see template_preprocess_field()
  */
 #}
 
-{% if label_hidden %}
-  {% if multiple %}
-    {% for item in items %}
-      {{ item.content }}
-    {% endfor %}
-  {% else %}
-    {% for item in items %}
-      {{ item.content }}
-    {% endfor %}
-  {% endif %}
-{% else %}
-  <div{{ attributes }}>
-    <div{{ title_attributes }}>{{ label }}</div>
-    {% if multiple %}
-      <div>
-    {% endif %}
-    {% for item in items %}
-      {{ item.content }}
-    {% endfor %}
-    {% if multiple %}
-      </div>
-    {% endif %}
-  </div>
-{% endif %}
+{% include "@cwd_biotech/includes/incl--field--image-video-embed.html.twig" %}
```

Comment:
I think `@cwd_biotech` should be replaced since this is on `cwd_base`? Does `../includes/incl--field--image-video-embed.html.twig` work? Also applies to the other file with an include.

Reply:
YES! Yikes haha -- thanks!